src/main/java/io/github/hildi/can/service/StandardJavaSerializationService.java

```diff
@@ -0,0 +1,48 @@
+package io.github.hildi.can.service;
+
+import io.github.hildi.can.exceptions.DeserializeException;
+import io.github.hildi.can.exceptions.FileIsNotExistsException;
+import io.github.hildi.can.exceptions.NotWritableFileException;
+import io.github.hildi.can.model.User;
+
+import java.io.*;
+
+/**
+ * Created by Serhii Hildi on 15.03.19.
+ */
+public class StandardJavaSerializationService implements SerializationService {
+
+    private User user;
+
+    @Override
+    public void serialize(User user, File file) {
+
+        if (!file.exists()) {
+            throw new FileIsNotExistsException("We cannot write data at file. File is not exists.");
+        }
+
+        if (!file.canWrite()){
+            throw new NotWritableFileException("We cannot write data at file. Is not a writable");
+        }
+
+        try (ObjectOutputStream out = new ObjectOutputStream(new FileOutputStream(file))) {
+            out.writeObject(user);
+        } catch (IOException e) {
+            System.out.println("We cannot write data at file. " + e.getMessage());
```

Comment:
look, here something went wrong with IO, but you haven't let know to the caller.

imagine the case, when you use this service in your app, calling this service method:
```
SerializationService service = new StandardJavaSerializationService();
service.serialize(user, file); // some io error arise inside 
```
you will never know if something goes wrong in that method, without looking to the system output.

Reply:
Sorry, I dont understand what are you mean.